FeedStoreChallenge/CoreDataManager/ManagedCache+CoreDataClass.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ManagedCache+CoreDataClass.swift
+//  FeedStoreChallenge
+//
+//  Created by Errol on 27/08/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedCache)
+public class ManagedCache: NSManagedObject {
```

Comment:
`ManagedCache` is an internal detail of `CoreDataFeedStore` thus it should not be public.
Also this class is not intended to be inherited thus it should be `final`
Please remove `public` modifier and make the class `final`.
```suggestion
final class ManagedCache: NSManagedObject {
```

Reply:
Done.